### ISSUE TITLE
Typeahead: suggested options not conveyed to screen readers

### ISSUE THREAD

**Issue by hvinh** [ng-bootstrap/ng-bootstrap#1945]
### Bug description:
Accessibility issue
As raised during an accessibility audit, when using an autocomplete feature, the list of suggested options should be conveyed to screen readers users.
The same issue is observed with the ng-bootstrap typeahead widget.
After typing a letter and options appear, this information is currently not conveyed to screen reader users, who can not know that options are available.
Same issue is occurring when no option is found.
The recommendation is to add a hidden sentence for screen readers like "2 options are available, please use up and down arrow keys to navigate"/"No results available" that will be read out after typing a letter.
An aria live region could be used to achieve this (role="status" and aria-live="polite").

Tested with NVDA/Firefox and JAWS/IE

### Link to minimally-working plunker that reproduces the issue:
https://ng-bootstrap.github.io/#/components/typeahead/examples

### Version of Angular, ng-bootstrap, and Bootstrap:
Angular: angular 4.4.6

ng-bootstrap: 1.0.0-beta.5

Bootstrap: 4.0.0-beta.2

**Comment by changLiuUNSW** [comment]
I have tested typeahead and it works perfect with JAW18 and IE edge.
Please note that different version of browsers and screen readers may have different behaviour for some aria attibutes, say the accessibility behsvior for Jaws 18 + IE 11 and Jaws18 + IE wedge are different.

**Comment by hvinh** [comment]
Hello,
The options can indeed be selected with a screen reader.
The problem is more that when the options appear after typing, there is no warning for the screen reader user, who therefore is not aware that options are available.

**Comment by changLiuUNSW** [comment]
Did you tested with JAW18 on IE eadge? Also did you test the aceesibility of Google search ?

**Comment by hvinh** [comment]
I tested but the version of the screen reader makes no difference as this behavior is not implemented.
Surprisingly it is not even implemented on the Google search.
